### PR TITLE
Press contact page

### PR DESCRIPTION
**Pull request by sebbsan**
@ajantsch structurally done. Just need your magic touch for the layout ;)

### CODE REVIEW

File: src/data/copy.json [hunk 1]
```diff
@@ -66,6 +66,18 @@
       ],
       "contact": "Für weitere Fragen und stellvertretenden Kontakt, kontaktieren sie bitte: Solveig Rathenow <solveig.rathenow@finleap.com> "
     },
+    "press": {
+      "title": "PR Kontakt",
+      "pageHeader": "Schreiben Sie Uns",
+      "pressContactName": "Solveigh Rathenow",
+      "pressContactTitle": "Head of Venture Communications",
+      "pressContactEmail": "solveig.rathenow@finleap.com",
+      "pressContactPhone": "+49 176 321 44 053",
```
Comment (manzke) [inline src/data/copy.json]: didn't we said we take it out?

File: src/data/copy.json [hunk 2]
```diff
@@ -146,6 +158,18 @@
       ],
       "contact": "Für weitere Fragen und stellvertretenden Kontakt, kontaktieren sie bitte: Solveig Rathenow <solveig.rathenow@finleap.com> "
     },
+    "press": {
+      "title": "Press Contact",
+      "pageHeader": "Reach out to us",
+      "pressContactName": "Solveigh Rathenow",
+      "pressContactTitle": "Head of Venture Communications",
+      "pressContactEmail": "solveig.rathenow@finleap.com",
+      "pressContactPhone": "+49 176 321 44 053",
```
Comment (manzke) [inline src/data/copy.json]: same

File: src/data/copy.json (hunk 1 above)
Comment (sebbsan) [reply src/data/copy.json]: right nobody told me. Taking phone number out

Comment (sebbsan) [reply src/data/copy.json]: done

**Comment by sebbsan** [comment]
@manzke ds_store file was removed

**Pull request merged**